src/main/java/edu/harvard/iq/dataverse/pidproviders/FakePidProviderServiceBean.java

```diff
@@ -0,0 +1,66 @@
+package edu.harvard.iq.dataverse.pidproviders;
+
+import edu.harvard.iq.dataverse.AbstractGlobalIdServiceBean;
+import edu.harvard.iq.dataverse.DvObject;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.ejb.Stateless;
+
+@Stateless
+public class FakePidProviderServiceBean extends AbstractGlobalIdServiceBean {
+
+    @Override
+    public boolean alreadyExists(DvObject dvo) throws Exception {
+        return true;
+    }
+
+    @Override
+    public boolean registerWhenPublished() {
+        return false;
```

Comment:
Maybe this should be a setting instead? It might be usefull during demos of the code.